### ISSUE TITLE
SNOW-464321: Parsing Json

### ISSUE THREAD

**Issue by 2020testuser** [snowflakedb/snowflake-connector-python#870]
## What is the current behavior?
Currently, the json in the variant column can only be parsed using json.parse() and this is slow. 
## What is the desired behavior?
Looking for a function that can parse (efficiently) the json string in the variant column 
## How would this improve `snowflake-connector-python`?
Better performance
 
Any better way to parse the json would be very helpful as my json is over 25,000 lines and hard to parse using json.parse(). Thanks in advance!

**Comment by sfc-gh-mkeller** [comment]
Hi @2020testuser 
Currently we don't support any JSON parsing in our Python library, so I have marked this issue as an enhancement.

From some googling around I found the following library https://github.com/ijl/orjson which promises to be the fastest Python JSON library. I have never used it myself, but please feel free to give it a try and let us know of how you find it, so that we can take your feedback into account when we add native JSON support to the connector

**Comment by sfc-gh-achandrasekaran** [comment]
To clean up and re-prioritize bugs and feature requests we are closing all issues older than 6 months as of March 1, 2023. If there are any issues or feature requests that you would like us to address, please re-create them. For urgent issues, opening a support case with this link [Snowflake Community](https://community.snowflake.com/s/article/How-To-Submit-a-Support-Case-in-Snowflake-Lodge) is the fastest way to get a response
